modules/persistence/src/commonMain/resources/db/migration/1.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE account (
+    id  SERIAL PRIMARY KEY,
+    username VARCHAR(255) UNIQUE NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP
+);
+
+CREATE INDEX account_username_index ON account (username);
+
+INSERT INTO account (username) VALUES ('root');
```

Comment:
Add trailing new line

Reply:
This is auto generated.

Reply:
Also thr trailing newlines are not that big of a thing. I rather have people focus on real issues with the code I'm a review